frontend/public/locales/en/search.json

```diff
@@ -51,5 +51,7 @@
     "search.saved.searches.error": "Query error related to saved search results.",
     "search.saved.searches.section.title": "Saved searches",
     "search.suggested.searches.seaction.title": "Suggested search templates",
-    "searchbar.button.save.search": "Save search"
+    "searchbar.button.save.search": "Save search",
+    "messages.S20.short": "More on disabled clusters",
+    "messages.S20.long": "Currently, search is disabled on some of your managed clusters. Some data might be missing from the console view. See ______ to enable search."
```

Comment:
> "See ______ to enable search."
- We need to add the link to the documentation.
- To move this PR forward, let's omit this sentence.

Reply:
do we have a link to enable search? I thought that was only done through editing the klusterletaddonconfig file.